dedicated-admin-operator-services/dedicated-admin-operator-operator.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+- hash: none
+  name: dedicated-admin-operator
+  path: /hack/olm-registry/olm-artifacts-template.yaml
```

Comment:
this file doesn't exist

Reply:
Updated the path reference to: `/build/templates/olm-artifacts-template.yaml.tmpl`
This template is added by the following PR: https://github.com/openshift/dedicated-admin-operator/pull/37